.github/workflows/update-bridge.yml

```diff
@@ -52,35 +47,33 @@ jobs:
       with:
         python-version: ${{matrix.pythonversion}}
     - run: sudo npm install -g chg
-    - name: Update Pulumi Terraform Bridge Dependency
-      run: cd provider && go mod edit -require github.com/pulumi/pulumi-terraform-bridge/v3@${{
-        github.event.client_payload.ref }} && go mod tidy && cd ../
+    - name: Update pulumi-terraform-bridge
+      run: cd provider && go mod edit -require github.com/pulumi/pulumi-terraform-bridge/v3@v${{ github.event.inputs.bridge_version }} && go mod tidy
+    - name: Update Pulumi SDK (provider/go.mod)
+      run: cd provider && go mod edit -require github.com/pulumi/pulumi/sdk/v3@v${{ github.event.inputs.sdk_version }} && go mod tidy
+    - name: Update Pulumi SDK (sdk/go.mod)
+      run: cd sdk && go mod edit -require github.com/pulumi/pulumi/sdk/v3@v${{ github.event.inputs.sdk_version }} && go mod tidy
+    - run: make tfgen
     - run: make build_sdks
-    - run: chg add "Upgrading pulumi-terraform-bridge to ${{ github.event.client_payload.ref
-        }}
     - name: commit changes
       uses: EndBug/add-and-commit@v7
```

Comment:
We should be able to combine these 2 steps using `https://github.com/peter-evans/create-pull-request`

Reply:
Fixed.